.github/workflows/package-apisix-dashboard-deb-ubuntu20.04.yml

```diff
@@ -54,3 +54,9 @@ jobs:
               exit 1
           fi
 
+      - name: Publish Artifact
```

Comment:
we could add `retention-days` for it.

Reply:
@nic-chen yep, 7 days is more proper. How about change this in another PR? :) Thanks.

Reply:
sounds good, we could add in another PR uniformly.